app/loader.h

```diff
@@ -89,6 +89,9 @@ class Loader: public QObject
     LayersModel &mLayersModel;
     AppSettings &mAppSettings;
 
+    //! Reloads project; if force us is true, reloads mProject on given path, otherwise used only for loading a new projects.
```

Comment:
typo - us?

```suggestion
    //! Reloads project; if force is true, reloads mProject on given path, otherwise used only for loading a new projects.
```

Reply:
![iu](https://user-images.githubusercontent.com/804608/84624277-5aa72700-aee1-11ea-82cd-6d2119d8c4ab.jpeg)

Reply:
@PeterPetrik You did that to me! :grin: